### PR TITLE
Refactorings to make the code more maintainable.

### PR DESCRIPTION
**Pull request by nisarhassan12**

### CODE REVIEW

File: src/components/pricing/Bussiness.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react'
```
Comment (JesterOrNot) [inline src/components/pricing/Bussiness.tsx]: The name of this file is spelled incorrectly, Buissiness -> Business

File: src/components/features/StartForFree.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```
Comment (JesterOrNot) [inline src/components/features/StartForFree.tsx]: Small nit-pick, maybe a simpler name for the component? Possibly StartFree

File: src/components/index/Banner.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
+import React from 'react'
+
```
Comment (JesterOrNot) [inline src/components/index/Banner.tsx]: There seems to be 2 Banner.tsx files one here and one in src/components, why is this?

Comment (nisarhassan12) [reply src/components/index/Banner.tsx]: Thanks! this one is specific to index page and the other one is used across various pages.

Comment (JesterOrNot) [reply src/components/index/Banner.tsx]: Got it

**Pull request merged**